lang/de.json

```diff
@@ -490,7 +490,7 @@
     "stats.volume": "Dein Reisevolumen",
     "stats.time": "Deine tägliche Reisezeit",
     "stats.per-week": "pro Kalenderwoche",
-    "stats.trips": "Fahrten",
+    "stats.trips": ":count Fahrt | :count Fahrten",
```

Comment:
I'm not sure about this one

Reply:
Is correctly translated into German. :)

But please remove the spaces between the variants (also in en & pl), then I can merge it.

Reply:
I removed the spacing 😄